### PR TITLE
docs: update ory permissions intro

### PR DESCRIPTION
**Pull request by tomekpapiernik**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/guides/permissions/overview.mdx [hunk 1]
```diff
@@ -146,7 +150,8 @@ You can create additional fine-grained permission rules for certain objects, sim
 <CodeFromRemote src={file("relation-tuples/tuples.json")} title="tuples.json" />
 ```
 
-Save the file as `tuples.json` in your current working directory. To load the file into Ory Permissions, run this command:
+Save the file as `tuples.json` in your current working directory. To create these permissions in Ory Permissions, run this
```
Comment (hperl) [inline docs/guides/permissions/overview.mdx:153]:
```suggestion
Save the file as `tuples.json` in your current working directory. To create these relationships in Ory Permissions, run this
```

File: docs/guides/permissions/overview.mdx [hunk 2]
```diff
@@ -165,7 +170,7 @@ ory create relationships tuples.json
 
 ### Checking for permissions
 
-Now, let's check some permissions! Some queries to try:
+With the permissions created, try running queries that illustrate use cases:
```
Comment (hperl) [inline docs/guides/permissions/overview.mdx:173]:
```suggestion
With the relationships created, try running queries that illustrate use cases:
```

File: docs/guides/permissions/overview.mdx [hunk 3]
```diff
@@ -198,7 +203,7 @@ ory is allowed User:Henning edit File private
 Allowed
 ```
 
-## Further reading
+## Next steps
 
-To learn more about the Ory Permission Language, read the [specification](../../keto/reference/ory-permission-language.mdx)
-document.
+- Learn [what is a permission model and how to create one](../../keto/modeling/create-permission-model.mdx).
```
Comment (hperl) [inline docs/guides/permissions/overview.mdx:208]:
```suggestion
- Learn [what a permission model is and how to create one](../../keto/modeling/create-permission-model.mdx).
```

Comment (hperl) [reply docs/guides/permissions/overview.mdx:208]: Should we motivate this some more to put the reader on the right track? Something like:

Now you know the basics of working with relationships and permissions in Ory Permissions. Next, make it yours by learning what a permission model is and how to create one.

WDYT?

Also what's really missing is how to integrate this into your app, but that's what @zepatrik is working on, right?

File: docs/keto/index.md [hunk 4]
```diff
@@ -1,45 +1,56 @@
...
+- Scalability: While traditional permission systems often struggle to handle large-scale applications and services with high
+  request volume, Ory Permissions is designed to be highly scalable and handle a large number of concurrent requests without
+  sacrificing performance or reliability.
 
-Ory Permissions operates on and manages relationships. These relationships represent the source of truth from which all
-permissions are derived. Common examples include:
+- Consistency: In older permissions systems, the data is often eventually consistent, which leaves windows of opportunity for
+  potential unauthorized access to resources before the data is correctly propagated to all parts of the system. By implementing
+  Zanzibar, Ory Permissions puts consistency in the spotlight and ensures that permission checks that are at the critical path of
+  user experience, are always answered correctly.
```
Comment (hperl) [inline docs/keto/index.md:29]: Without Zookies, we also only have eventual consistency. So we can't really claim that yet.

File: docs/keto/index.md [hunk 5]
```diff
@@ -1,45 +1,56 @@
...
 
-- `User c is the owner of Document d`
-- `User a is a member of Group b`
+With Ory Permissions, you can:
 
-In most cases relationships represent a fact that is already part of your application. To ensure the Ory Permission Server always
-has the most up-to-date information, you should use it as the source of truth for your application as well. There are multiple
-APIs to query and manage relationships.
+- Use Ory Permission Language to create permission models that fit your exact use case - RBAC, ABAC, and beyond.
```
Comment (hperl) [inline docs/keto/index.md:33]:
```suggestion
- Use the Ory Permission Language to create permission models that fit your exact use case - RBAC, ABAC, and beyond.
```

File: docs/keto/index.md [hunk 6]
```diff
@@ -1,45 +1,56 @@
...
+- Unify authorization logic in one service that's the single source of truth for access rights across all of your applications.
+- Be tech-stack agnostic and use Ory Network [SDKs](./sdk/01_overview.md) available for all major programming languages.
+- Issue fine-grained permissions, for example `user x can read document y`.
```
Comment (hperl) [inline docs/keto/index.md:36]:
```suggestion
- Issue fine-grained permissions, for example `User:x is in readers of Document:y`.
```

File: docs/keto/index.md [hunk 7]
```diff
@@ -1,45 +1,56 @@
...
+- Allow permissions inheritance through groups, roles, and hierarchies to ensure organic scaling that follows the growth of your
+  application.
 
-### Ory Permission Language
+## Ory Permission Language
 
-The Ory Permission Language (OPL) defines the rules by which permissions are derived from relationships. Every application has
-different rules, so this allows you to customize the behavior to your permission model. Examples of rules are:
+The Ory Permission Language (OPL) is a developer-friendly, [TypeScript](https://www.typescriptlang.org/)-based configuration
+language designed to have a low learning curve and a familiar source in contrary to other, proprietary languages used to represent
```
Comment (hperl) [inline docs/keto/index.md:43]:
```suggestion
language designed to be easy to learn and have a familiar source in contrary to other, proprietary languages used to represent
```

Comment (hperl) [reply docs/keto/index.md:43]: Learning curve metaphors are really broken IMHO, as a steep curve is actually a good thing (you learn things quickly), but has a bad connotation.

**Review by hperl — CHANGES_REQUESTED** [review]
This looks really good already, you got all the concepts right! I added some remarks below.

File: docs/keto/index.md (hunk 7 above)
Comment (zepatrik) [inline docs/keto/index.md:43]:
```suggestion
The Ory Permission Language (OPL) is a developer-friendly configuration language defined as a subset of [TypeScript](https://www.typescriptlang.org/).
It is designed to have a low learning curve and a familiar source in contrary to other, proprietary languages used to represent
```

Comment (tomekpapiernik) [reply docs/keto/index.md]: Updated with a version that mixes your and Patrik's suggestion :)

File: docs/guides/permissions/overview.mdx [hunk 8]
```diff
@@ -146,7 +150,8 @@ You can create additional fine-grained permission rules for certain objects, sim
 <CodeFromRemote src={file("relation-tuples/tuples.json")} title="tuples.json" />
 ```
 
-Save the file as `tuples.json` in your current working directory. To load the file into Ory Permissions, run this command:
+Save the file as `tuples.json` in your current working directory. To create these permissions in Ory Permissions, run this
+command:
 
 ```shell
 ory create relationships tuples.json
```
[the remark this answers is not in the document]
Comment (tomekpapiernik) [reply docs/guides/permissions/overview.mdx]: applied

File: docs/keto/index.md [hunk 9]
```diff
@@ -1,45 +1,56 @@
...
+described in [Zanzibar: Google's Consistent, Global Authorization System](https://research.google/pubs/pub48190/).
 
-- unify authorization logic in one service that is the authoritative source across all your applications.
-- define a permission model such as RBAC, ABAC, or a more flexible model that fits your use case.
-- work independent of any tech-stack, with [SDKs](./sdk/01_overview.md) available for all major programming languages.
-- query permissions across the world against the globally-distributed Ory Network.
-- issue fine-grained permissions, for example `user x can read document y`.
-- inherit permissions through groups, roles, hierarchies, and more.
+Implementing the design principles described in the Zanzibar paper allows Ory Permissions to offer the following benefits over
+older, widely adopted permission systems:
```
[the remark this answers is not in the document]
Comment (zepatrik) [reply docs/keto/index.md:15]: Update it here?

[the remark this answers is not in the document]
Comment (tomekpapiernik) [reply docs/keto/index.md]: Missed it initially. Done now :)

**Review by zepatrik — APPROVED** [review]
:pray: very nice

**Pull request merged**